internal/outputs/infra.go

```diff
@@ -76,5 +76,8 @@ func createStorer() (persist.Storer, error) {
 	if os.Getenv("VERBOSE") == "true" || os.Getenv("VERBOSE") == "1" {
 		fmt.Fprintf(os.Stderr, "custom storer name: %s and TTL: %s\n", storerName, ttl)
 	}
-	return persist.NewFileStore(persist.DefaultPath(storerName), load.Logrus, ttl)
+
+	tempDir := os.Getenv("TEMP_DIR")
```

Comment:
If TEMP_DIR is undefined, will it use the old default storer path?

Reply:
Yes, if it's undefined the call to  TmpPath will call to DefaultPath.